### PR TITLE
contributing: Optimize commands, text in release procedure

### PR DESCRIPTION
**Pull request by wenzeslaus**
Update release procedure after 8.2.0 (that's right).

- Add cron job update
- Restructure updates to website and wiki
- Remove Launchpad notes (I think the reason was they are not used, at least not by the release manager.)
- Change of procedure for the mailing list announcements
- Improves precision in the text

### CODE REVIEW

File: doc/howto_release.md [hunk 1]
```diff
@@ -429,18 +441,16 @@ Software pages:
...
+If final release, publish related announcement press release at:
+
 - Our main mailing lists:
-    - <https://lists.osgeo.org/mailman/listinfo/grass-announce> | <grass-announce@lists.osgeo.org>
+    - <https://lists.osgeo.org/mailman/listinfo/grass-announce> | <grass-announce@lists.osgeo.org> (who can post here?)
     - <https://lists.osgeo.org/mailman/listinfo/grass-dev> | <grass-dev@lists.osgeo.org>
     - <https://lists.osgeo.org/mailman/listinfo/grass-user> | <grass-user@lists.osgeo.org>
-- FreeGIS: <freegis-list@intevation.de>
-- OSGeo.org: <news_item@osgeo.org>, <info@osgeo.org>
+- FreeGIS: <freegis-list@intevation.de> (we need procedure for local mailing lists)
```
Comment (neteler) [inline doc/howto_release.md:452]: Let's just drop this one.

File: doc/howto_release.md [hunk 2]
```diff
@@ -429,18 +441,16 @@ Software pages:
...
+- OSGeo.org: <news_item@osgeo.org>, <info@osgeo.org> (who can post here?)
```
Comment (neteler) [inline doc/howto_release.md:453]:
```suggestion
- OSGeo.org: <news_item@osgeo.org>, <info@osgeo.org> (shortened release email to be sent by release manager)
```

File: doc/howto_release.md [hunk 3]
```diff
@@ -429,18 +441,16 @@ Software pages:
...
-- If official release:
-    - publish related announcement press release at:
-- Our GRASS web site: /announces/
-    - Note: DON'T use relative links there
+
+If final release, publish related announcement press release at:
+
 - Our main mailing lists:
-    - <https://lists.osgeo.org/mailman/listinfo/grass-announce> | <grass-announce@lists.osgeo.org>
+    - <https://lists.osgeo.org/mailman/listinfo/grass-announce> | <grass-announce@lists.osgeo.org> (who can post here?)
```
Comment (neteler) [inline doc/howto_release.md:449]:
```suggestion
    - <https://lists.osgeo.org/mailman/listinfo/grass-announce> | <grass-announce@lists.osgeo.org> (shortened release email to be sent by release manager)
```

File: doc/howto_release.md [hunk 4]
```diff
@@ -429,18 +443,16 @@ Software pages:
...
     - <https://lists.osgeo.org/mailman/listinfo/grass-dev> | <grass-dev@lists.osgeo.org>
     - <https://lists.osgeo.org/mailman/listinfo/grass-user> | <grass-user@lists.osgeo.org>
-- FreeGIS: <freegis-list@intevation.de>
-- OSGeo.org: <news_item@osgeo.org>, <info@osgeo.org>
+- FreeGIS: <freegis-list@intevation.de> (we need procedure for local mailing lists)
```
Comment (wenzeslaus) [inline doc/howto_release.md:454]:
```suggestion
```

File: doc/howto_release.md [hunk 5]
```diff
@@ -321,6 +324,23 @@ ssh $USER@$SERVER1 "cd $SERVER1DIR ; ln -s grass-$VERSION.tar.md5sum grass-$MAJO
...
+sudo vim /etc/apache2/sites-enabled/000-default.conf`
+```
+
+Load the new configuration:
+
+```bash
+sudo systemctl reload apache2`
+```
+
+For new branches: Udpate `grass-devel` using the steps above.
```
Comment (neteler) [inline doc/howto_release.md:342]:
```suggestion
For new branches: Update `grass-devel` using the steps above.
```

**Comment by neteler** [comment]
Let me suggest to continue this PR in #2414 (and related PRs for G83 and G82).

**Comment by wenzeslaus** [comment]
I "melded" the current file from main with the file here, updated from main with `git merge upstream/main -X theirs` and then used my "melded" file. This now should include all the originally proposed updates (even those which were not in #3030).

**Comment by landam** [comment]
@neteler Please consider approving this PR.

**Review by neteler — APPROVED** [review]

**Pull request merged**